src/fides/api/models/privacy_preference.py

```diff
@@ -153,6 +166,12 @@ def __tablename__(self) -> str:
 
     fides_string = Column(String)
 
+    property_id = Column(
+        String,
+        index=True,
```

Comment:
I'd remove `index=True` - I don't think you're adding this index in the migration, and we probably don't want want/need the overhead anyway.